src/addons/par/icc/rules.mk

```diff
@@ -75,4 +75,4 @@ $(OBJ_PAR_DIR)/generated/$(PAR_TOP).macros.out: $(OBJ_PAR_DIR)/synopsys-icc-macr
 $(OBJ_PAR_DIR)/generated/$(PAR_TOP).floorplan.json: \
 		$(PAR_TOOL_ADDON)/tools/generate-floorplan-json \
 		$(OBJ_PAR_DIR)/generated/$(PAR_TOP).macros.out
-	$(abspath $<) --macros $(filter %.macros.out,$^) --rtl_top $(SYN_TOP) --config $(PAR_CONFIG) -o $@
+	PATH="$(abspath $(dir $(CMD_PYTHON3))):$(PATH)" $(abspath $<) --macros $(filter %.macros.out,$^) --rtl_top $(SYN_TOP) --config $(PAR_CONFIG) -o $@
```

Comment:
Yeah so if there is no problem with just always using ```CMD_PYTHON3``` I don't see why we don't just set ```PATH``` that way at the top-level makefile to reduce this code duplication.